src/plugins/data/common/search/aggs/agg_config.ts

```diff
@@ -316,9 +316,9 @@ export class AggConfig {
   }
 
   /**
-   * @returns Returns an ExpressionAst representing the function for this agg type.
+   * @returns Returns an ExpressionAst representing the this agg type.
    */
-  toExpressionAst(): ExpressionAstFunction | undefined {
+  toExpressionAst(): ExpressionAstExpression | undefined {
```

Comment:
Discovered that returning an `ExpressionAstFunction` was not very useful since using this as a subexpression requires it to be `ExpressionAstExpression`, so I updated accordingly.